### ISSUE TITLE
Add CORS policy for S3

### ISSUE THREAD

**Issue by lognaturel** [getodk/docs#1906]
Web Forms will directly fetch from S3

Bucket -> Permissions -> Cross-origin resource sharing (CORS)
```
[
    {
        "AllowedHeaders": [],
        "AllowedMethods": [
            "GET"
        ],
        "AllowedOrigins": [
            "https://my.website"
        ],
        "ExposeHeaders": []
    }
]
```

**Comment by alxndrsn** [comment]
Was this fixed by https://github.com/getodk/web-forms/issues/859?

**Comment by lognaturel** [comment]
No, we'll always need a CORS config. We addressed https://github.com/getodk/web-forms/issues/859 by removing the extra headers so that a minimal policy like the one documented here works. Alternatively we could have added the headers that were being added to the policy.
